### PR TITLE
Add addtional email field support in backend

### PR DESCRIPTION
**Pull request by erlingfn**
Add support for people outside of abakus to be receive mails from email lists together with webkom/lego-webapp#1606

### CODE REVIEW

**Comment by erlingfn** [comment]
> Looks good, just add a test or two for some weird cases like using additional emails as well as requiring internal emails.

I believe this is case is already covered by the "test_only_internal_allowed" test in "test_models"

File: lego/apps/email/tests/test_views.py [hunk 1]
```diff
@@ -74,6 +77,68 @@ def test_create_list_invalid_email(self):
...
+            [
+                "test1@user.com",
+                "user@admin.com",
+                "abakusgroup@admin.com",
+                "additional@email.com",
+            ],
+            members,
+        )
+
+    @skip
```
[the remark this answers is not in the document]
Comment (odinuge) [reply lego/apps/email/tests/test_views.py]: ref. the slack post I guess

File: lego/apps/email/tests/test_views.py [hunk 2]
```diff
@@ -74,6 +77,68 @@ def test_create_list_invalid_email(self):
...
+                "name": "Invalid",
+                "email": "valid",
+                "users": [1, 2],
+                "group_roles": ["member"],
+                "additional_emails": ["invalid"],
+            },
+        )
+        self.assertEqual(status.HTTP_400_BAD_REQUEST, response.status_code)
+
+    def test_edit_additional_email(self):
```
[the remark this answers is not in the document]
Comment (erlingfn) [reply lego/apps/email/tests/test_views.py]: Good idea, I will add a test for that!

File: lego/apps/email/tests/test_views.py [hunk 3]
```diff
@@ -74,6 +77,68 @@ def test_create_list_invalid_email(self):
...
+                "group_roles": ["member"],
+                "additional_emails": ["additional@email.com", "additional@email.com"],
+            },
+        )
+        self.assertEqual(status.HTTP_201_CREATED, response.status_code)
+
+        email_list = EmailList.objects.get(email="jubileum")
+        members = email_list.members()
+
+        self.assertCountEqual(
```
[the remark this answers is not in the document]
Comment (erlingfn) [reply lego/apps/email/tests/test_views.py]: Does the order matter in this context? Although duplicates are tested elsewhere, converting both to set would remove that check from the test

[the remark this answers is not in the document]
Comment (orhanhenrik) [reply lego/apps/email/tests/test_views.py]: You're right 👍

File: lego/apps/email/tests/test_views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (erlingfn) [reply lego/apps/email/tests/test_views.py]: Removed the skip thanks to #1663

**Comment by erlingfn** [comment]
> Nice.
> What happens if you add an additional email that is the same as a users email (who is included in the email list)?

Duplicates are removed at the end of the members method, so members will only contain one copy of the email.

**Pull request merged**